apps/dashboard/src/components/RightPanel/FontSection.tsx

```diff
@@ -70,7 +70,12 @@ export function FontSection({ selectedElement }: FontSectionProps) {
         }} type="number" value={selectedElement.lineHeight}>
           <LineHeightIcon />
         </Input>
-        <Input max={5} min={0} onChange={() => { updateElement(selectedElement); }} type="number" value={0}>
+        <Input max={5} min={0} onChange={value => {
```

Comment:
Let's also allow going down to `-10` and up to 10, just to allow more customization.

Reply:
Yea, I'll make it works now.